exonum-node/src/schema.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use exonum::{
+    crypto::PublicKey,
+    helpers::Round,
+    merkledb::{
+        access::{Access, AccessExt, RawAccessMut},
+        ListIndex, MapIndex,
+    },
+    messages::Verified,
+};
+
+use std::iter;
+
+use crate::messages::{Connect, Message};
+
+const CONSENSUS_MESSAGES_CACHE: &str = "core.consensus_messages_cache";
```

Comment:
Would it make sense to prepend node indexes with `node.` instead of `core.`?

Reply:
I went for backward compatibility, which sort of makes sense for peers cache (not so much for other two indexes).